testing/test_pdb.py

```diff
@@ -75,6 +75,21 @@ def test_1():
         if child.isalive():
             child.wait()
 
+    def test_pdb_interaction_capture(self, testdir):
+        p1 = testdir.makepyfile("""
+            def test_1():
+                print("getrekt")
+                assert False
+        """)
+        child = testdir.spawn_pytest("--pdb %s" % p1)
+        child.expect("(Pdb)")
```

Comment:
Oh actually, the output is displayed after PDB, so this is definitely a bug. Let me see if I can fix.